docs/en/latest/apisix-dashboard.md

```diff
@@ -31,10 +31,38 @@ $ helm repo update
 $ helm install apisix-dashboard apisix/apisix-dashboard
 ```
 
+- Configure the custom plug-in JSONSchema [How to get jsonschema?](https://apisix.apache.org/docs/dashboard/FAQ/#4-after-modifying-the-plugin-schema-or-creating-a-custom-plugin-in-apache-apisix-why-cant-i-find-it-on-the-dashboard)
+
+```shell
+helm install <name> apisix/apisix-dashboard . -n <namespace> --set-file config.schema=./schema.json
+```
+
 ## Uninstall
 
 To uninstall/delete the `apisix-dashboard` release:
 
 ```shell
 $ helm uninstall apisix-dashboard
 ```
+
+## Configuration
+
+Parameter | Description | Default
+------------ | ------------- | ------------- 
+ Config.conf |  |  
```

Comment:
`config`?

Reply:
Here is a separator to separate configuration items